src/renderer/atlas/AtlasEngine.api.cpp

```diff
@@ -161,6 +162,9 @@ constexpr HRESULT vec2_narrow(U x, U y, AtlasEngine::vec2<T>& out) noexcept
 
 [[nodiscard]] HRESULT AtlasEngine::UpdateSoftFont(const std::span<const uint16_t> bitPattern, const til::size cellSize, const size_t centeringHint) noexcept
 {
+    const auto softFont = _api.s.write()->font.write();
+    softFont->softFontPattern = std::vector(bitPattern.begin(), bitPattern.end());

```

Comment:
For these situations we could make `Buffer` a class that is reference counting, so that copying such buffers is as cheap as a AddRef/Release. Yay or nay?

Reply:
> Soft font updates should be rare, I'm OK with copying today.

Yep that's exactly how I saw it as well.

> Want to file a followup?

Yep!